02-getting_started.Rmd

```diff
@@ -1,67 +1,249 @@
+    $TOP_SRCDIR/configure  --enable-R-shlib
+    make 
+    make check
+
+Once you successfully build R from source you can modify R source code to fix an issue: Prepare a patch (See [this guide](https://www.r-project.org/bugs.html#how-to-submit-patches)) and after checking that R works as intended (`make check-devel`) submit the patch for the R core consideration.
+(See the [lifecycle of a patch](#FixBug) chapter).
+
+
+To use the r-devel version in RStudio to you can do the following:
+
+    export RSTUIDO_WHICH_R="$BUILDDIR/bin/R"
+    cd $TOP_SRCDIR
+    rstudio
+
+
+### Windows
+
+#### Binaries
+
+The latest documentation about installing R and packages from source at the current time can be find [here](https://cran.r-project.org/bin/windows/base/howto-R-4.2.html).
+
+1.  The binary builds of R for Windows can be downloaded and installed from [here](https://cran.r-project.org/bin/windows/base/).
+    Along with the link to the latest stable release, this page also contains links to the binary builds of `r-patched` and `r-devel`.
+
+2.  Click on the download links to download an executable installer.
+
+3.  Select the language while installing, read the GNU general public license information, and select the destination location to start the installation.
+    You will be prompted to select components at this stage: `User installation`, `64-bit User installation`, or `Custom installation`.
+    The default option may be chosen for the questions from this step onwards to complete the installation.
+    
+You can also use the [binaries for `r-devel` made available](https://cran.r-project.org/bin/windows/base/rdevel.html) which are not so up to date. 
+
+#### From source
+
+You can download the R sources on Windows <https://svn.r-project.org/R/>. 
+But if you want to keep up with changes it is recommended to checkout the sources from svn, it is probably best to install an SVN client.
+Either TortoiseSVN (<https://tortoisesvn.net/>, command line tool, and Windows Explorer integration) or SlikSVN (<https://sliksvn.com/download/>, just the command line tool) is recommended.
+
+1.  Open the Rtools42 terminal, and update and get the tools:
+
+      pacman -Syuu
+      pacman -Sy wget subversion
+      
+2. Download the latest tcl/tk
+
+      wget -np -nd -r -l1 -A 'tcltk-*.zip' https://cran.r-project.org/bin/windows/Rtools/rtools42/files/
+      
+3. Set environment variables for POSIX, MiKTeX, tar (and one tar option):
+
+        export PATH=/x86_64-w64-mingw32.static.posix/bin:$PATH
+        export PATH=/c/Program\ Files/MiKTeX/miktex/bin/x64:$PATH
+        export TAR="/usr/bin/tar"
+        export TAR_OPTIONS="--force-local"
+
+4. Check that all the programs can be found:
+
+        which make gcc pdflatex tar
+        
+    If there is any error you'll need to find where the program is installed and add the corresponding path.
+1.  Download the latest recommended packages[^02-getting_started-2]:
+
+        $TOP_SRCDIR/tools/rsync-recommended
```

Comment:
I tested the instructions on a user account named "Test account" - this caused issues with the use of variables because of the space in the username. This can be fixed by using quotes.
```suggestion
        "$TOP_SRCDIR/tools/rsync-recommended"
```
This applies to all uses of `$TOP_SRCDIR` and `$BUILDDIR`.